### PR TITLE
Added getCompany method and companyList variable

### PR DESCRIPTION
**Pull request by dostoevskoy**
According to issue #2 a new method getCompany has been added to the Iex class.

### CODE REVIEW

File: datawell/iex.py [hunk 1]
```diff
@@ -56,3 +57,28 @@ def load_from_iex(self, uri: str):
             error = response.status_code
             self.Logger.error(
                 f'Encountered an error: {error} ( {response.text} ) while retrieving {uri}')
+    
+    def getCompany(self, *Symbols):
```
Comment (petelind) [inline datawell/iex.py]: Make sure to explain * and ** to your colleagues, ok?

File: datawell/iex.py [hunk 2]
```diff
@@ -56,3 +57,28 @@ def load_from_iex(self, uri: str):
...
+        """
+        Will return companies according to the stocks or all stocks.
+        https://github.com/petelind/ConsumingAPI/issues/2
+        TODO:  ADD Cache for companies
+        TODO:  Add special symbol handler(*,-,+ etc.)
+        TODO:  ADD multithreads (takes 1h40m without)
```
Comment (petelind) [inline datawell/iex.py]: All ideas very good, but I'd suggest you leave TODOs out of the master codes - we typically employ those freely, but it seems like the general convention they are for the "work in progress". (If you'll have a look on the linter output and commit checks in PyCharm, you'll notice that they are frowning on the TODOs in the commit).

File: datawell/iex.py [hunk 3]
```diff
@@ -56,3 +57,28 @@ def load_from_iex(self, uri: str):
...
+        """
+        #make an unique set of Symbols
+        Symbols=set(Symbols) 
+        #if noone symbol is not provided, default set of symbols(all) will be used
+        if len(Symbols) == 0:
+            Symbols=[symbol['symbol'] for symbol in self.Symbols]
```
Comment (petelind) [inline datawell/iex.py]: Comprehensions! Excellent!

File: datawell/iex.py [hunk 4]
```diff
@@ -16,6 +16,7 @@ def __init__(self):
         self.Symbols = self.get_stocks()
         datapoints = ['logo', 'company']
         self.Datapoints = dict(zip(datapoints, datapoints))
+        self.companyList=[]
```
Comment (petelind) [inline datawell/iex.py]: JS haunts you, does it not? :)
Pythonic convention will be:
.PublicProperty
._private_field
.local_var

File: datawell/iex.py (hunk 1 above)
Comment (dostoevskoy) [reply datawell/iex.py]: sure!

**Comment by petelind** [comment]
@dostoevskoy, you are not diverging yet but will start soon...

**Comment by dostoevskoy** [comment]
All changes will be applied with a new pull request.

**Pull request closed without merging**